eslint.config.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const config = require('./index.js');
+
+module.exports = [
+	...config,
```

Comment:
[suggestion] I'm wondering if testing `jest.js` in the original `.eslintrc.js` might still be valuable. What do you think?

```suggestion
const config = require('./index.js');
const jestConfig = require('./jest.js');

module.exports = [
	...config,
	...jestConfig,
```

Reply:
We use `eslint.config.js` to configure ESLint for the repo, i.e. to lint the configs and their tests. The tests use `node:test` rather than Jest. Perhaps we accidentally left in the jest config in the original `.eslintrc` when we converted the tests to use `node:test`?

Reply:
Ah, got it. We've already tested `jest.js` in `__tests__/jest.test.js`, so my suggestion is unnecessary. 👍🏼